src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
> Are .preventDefault(), .stopPropagation(), and .stopImmediatePropagation() defining properties of events?

I'd say yes.

> Must every event.originalEvent. ... .originalEvent chain end in a native event?

Nope. But not because it can end in an empty object right now. It's because you can call `jQuery.Event()` with any event, even one that started from jQuery.